_includes/releases/v22.2/v22.2.0-alpha.1.md

```diff
@@ -0,0 +1,851 @@
+- Creating a table with a locality of [`REGIONAL BY ROW`](../{{site.versions["dev"]}}/multiregion-overview.html#regional-by-row-tables) could intermittently fail with a missing type error. [#80590][#80590]
+- [`SHOW EXPERIMENTAL_FINGERPRINTS FROM TABLE`](../{{site.versions["dev"]}}/experimental-features.html#show-table-fingerprints) now works on tables with partial indexes. [#80539][#80539]
+- Fixed a rare race condition that could allow for a transaction to serve a stale read and violate real-time ordering under moderate clock skew. [^1]: see [pkg/kv/kvserver/observedts/doc.go](../{{site.versions["dev"]}}/https://github.com/cockroachdb/cockroach/blob/master/pkg/kv/kvserver/observedts/doc.go) for an explanation of the role of observed timestamps in the transaction model. This commit updates that documentation to include this fix. [^2]: see analysis in https://github.com/cockroachdb/cockroach/issues/36431#issuecomment-714221846. [#80706][#80706]
+- The hex encoding for [`BYTEA`](../{{site.versions["dev"]}}/bytes.html) values now works properly when used in [`COPY FROM ... CSV`](../{{site.versions["dev"]}}/copy-from.html) statements. [#81120][#81120]
+- Constants in SQL query fields are correctly removed for `VIEWACTIVITYREDACTED` users. [#80707][#80707]
+- Fixed a gap in disk-stall detection. Previously, disk stalls during filesystem metadata operations could go undetected, inducing deadlocks. Now stalls during these types of operations will correctly fatal the process. [#81389][#81389]
+- Fixed a bug that caused duplicated schema change job description messages. [#81268][#81268]
+- Fixed false negatives produced by the `JSON` `?` operator when invoked on a `JSON` array with the vectorized engine set explicitly to `off`. [#81648][#81648]
+- Fixed a bug where [sequences](../{{site.versions["dev"]}}/create-sequence.html) could return values that are out-of-bounds in some cases. [#81123][#81123]
+- Fixed the formatting of floats in arrays and tuples sent over the client-server pgwire protocol so that they respect the `extra_float_digits` parameter, and correctly format infinity values. [#82022][#82022]
+- The [`DateStyle`](../{{site.versions["dev"]}}/show-vars.html#supported-variables) session setting is no longer ignored using the CLI when set in the `options` URL parameter. [#82101][#82101]
+- Dropping tables with foreign key dependencies would generate the wrong `pgcode` (`Uncategorized` versus `DependentObjectsStillExist`). This is now fixed. [#80142][#80142]
+- Previously, if a [foreign key](../{{site.versions["dev"]}}/foreign-key.html) was concurrently added while the referenced table was dropped before validation was completed, CockroachDB could potentially hang on the rollback. Fixed this by modifying the foreign key logic to generate appropriate errors when the referenced table is dropped and to gracefully rollback the change. [#80142][#80142]
```

Comment:
Fixed this by ... --> Now, CockroachDB will generate appropriate errors when the referenced table is dropped and gracefully rollback the change.